src/test/java/elizCurtnazar/tests/tema8/teste/BaseTest.java

```diff
@@ -0,0 +1,40 @@
+package elizCurtnazar.tests.tema8.teste;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import teofilursan.driver.BrowserManager;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BaseTest {
+
+    protected ChromeDriver driver;
+
+    @BeforeMethod(groups = "desktop")
+    public void initChromeBrowser() {
+        driver = BrowserManager.createChromeDriver();
```

Comment:
Call method for creating chrome driver maximized from BrowserManager class and remove the other unnecessary lines from this method. Thanks!

Reply:
done